frontend/lib/app.dart

```diff
@@ -23,6 +23,52 @@ const homeRouteParamTabIndexName = 'tabIndex';
 const homeRouteName = '/home';
 const introRouteName = '/intro';
 
+final GoRouter router = GoRouter(
+  routes: <RouteBase>[
+    GoRoute(
+      path: initialRouteName,
+      builder: (BuildContext context, GoRouterState state) {
+        final settings = Provider.of<SettingsModel>(context);
+        return settings.firstStart
+            ? IntroScreen(
+                onFinishedCallback: () => settings.setFirstStart(enabled: false),
+              )
+            : HomePage();
+      },
+      routes: [
+        GoRoute(
+          path: '$activationRouteName/:$activationRouteCodeParamName',
+          builder: (BuildContext context, GoRouterState state) {
+            print(state.uri.fragment);
+            return DeepLinkActivation(base64qrcode: Uri.decodeFull(state.uri.fragment));
+          },
+        ),
+      ],
+    ),
+    GoRoute(
+      path: homeRouteName,
+      builder: (BuildContext context, GoRouterState state) {
+        return HomePage(initialTabIndex: int.parse(state.pathParameters[homeRouteParamTabIndexName]!));
+      },
+    ),
```

Comment:
Is this actually necessary?

Reply:
I removed it again @f1sh1918 as I think it is wrong 🤔

Reply:
Yes It is necassary. I described this in my proposal @michael.markl
Just passing an index is not needed my fault. Just forward to the Plain HomePage()

Reply:
If you confirm the intro slides there will be a no route found for home exception. please add 
```js
    GoRoute(
      path: homeRouteName,
      builder: (BuildContext context, GoRouterState state) {
        return HomePage();
      },
    )
```

Reply:
Ah sorry, I didn't see your proposal PR. It's still a bit weird, that we need this route... Especially as HomePage now has three routes...
I added the route again.

Reply:
yes well the first is the inititalRoute which depends on settings and then we have a home route and a home route with index. Probably we could also remove that route and just redirect to to home/:0 for the map route `onFinishIntroCallback`
Maybe it also makes sense to use TabNames instead of index, because if we add a new tab between we will get redirected to the wrong tab with a given index.